lib/app.js

```diff
@@ -5,6 +5,7 @@ const express = require('express');
 const config = require('./config');
 const SioFb = require('./sioFb');
 const SioBot = require('./sioBot');
+const pkg = require('./../package.json');
```

Comment:
I am used to putting a `./` when requiring local files. If this is moved to the main directory it would refactor to `require('./package.json')`, which is correct, whereas `require('package.json')` is not.